code/main/edge/edgeModule.brs

```diff
@@ -35,31 +35,27 @@ function _adb_EdgeModule(configurationModule as object, identityModule as object
         _identityModule: identityModule,
         _edgeRequestWorker: _adb_EdgeRequestWorker(),
 
-        processEvent: function(requestId as string, eventData as object, timestampInMillis as longinteger) as dynamic
+        ''' sendEvent API triggers this API to queue edge requests
```

Comment:
[optional] can we use a single `'`  for comments? I believe it is clearer than `'''` and it is also the BrightScript's standard.
https://developer.roku.com/docs/references/brightscript/language/expressions-variables-types.md#in-brightscript

```
' ****************************************************************************************************
'
' Before calling this function to send a Media event with the given XDM data, it's required to call the
' "createMediaSession" API to start a new session.
'
' @param xdmData as object : the XDM data of the Media event
'
' ****************************************************************************************************
```